code/modules/events/flutes.dm

```diff
@@ -0,0 +1,142 @@
+//This gets executed at the start of the round
+/datum/round_event/flutes/start()
+	var/list/remaining_players = list_intersection(GLOB.player_list, GLOB.alive_mob_list)
+	var/number_of_players_to_select = round((remaining_players.len / 10), 1)
+	for(var/i; i<=(number_of_players_to_select == 0 ? 1 : number_of_players_to_select); i++)
+		var/player = remaining_players[rand(1, remaining_players.len)]
+		sound_intro(player)
+		pick_flute_scene(player)
+		remaining_players.Remove(player)
+		chosen_players.Add(player)
+
+//EVENT PROCS
+/datum/round_event/flutes/proc/sound_intro(mob/living/carbon/M)
+	if((M.client.prefs.toggles & SOUND_MIDI) && is_station_level(M.loc.z))
+		M.playsound_local(M, 'sound/ambience/flutes.ogg', 20, FALSE, pressure_affected = FALSE)
+
+/datum/round_event/flutes/proc/pick_flute_scene(mob/living/carbon/M)
+	var/flutes_chosen_scene = "abc"
+	switch(rand(1, 6))
+		if(1)
+			flutes_chosen_scene = "flute_vis_flicker"
+			flute_vis_flicker(M)
+		if(2)
+			flutes_chosen_scene = "flute_headache"
+			flute_headache(M)
+		if(3)
+			flutes_chosen_scene = "flute_tremble"
+			flute_tremble(M)
+		if(4)
+			flutes_chosen_scene = "flute_chanting"
+			flute_chanting(M)
+		if(5)
+			flutes_chosen_scene = "flute_starlight"
+			flute_starlight(M)
+		if(6)
+			flutes_chosen_scene = "flute_corridor_encounter"
+			flute_corridor_encounter(M)
+
+/datum/round_event/flutes/proc/flute_vis_flicker(mob/living/carbon/M)
+	to_chat(M, "<span class='warning'><b>Your vision flickers.</b></span>")
+	M.blur_eyes(15)
+
+/datum/round_event/flutes/proc/flute_headache(mob/living/carbon/M)
+	to_chat(M, "<span class='warning'><b>You get an intense headache!</b></span>")
+	M.blur_eyes(15)
+	M.adjustOrganLoss(ORGAN_SLOT_BRAIN, 15, 20, 30)
+	M.adjustStaminaLoss(15)
+
+/datum/round_event/flutes/proc/flute_tremble(mob/living/carbon/M)
+	to_chat(M, "<span class='warning'><b>Something trembles along the edge of your vision, your eyes water, with the familiar beat of blood racing through your head.</b></span>")
+	M.blur_eyes(30)
+	M.adjustOrganLoss(ORGAN_SLOT_BRAIN, 30, 35)
+	M.adjustStaminaLoss(30)
+
+/datum/round_event/flutes/proc/flute_chanting(mob/living/carbon/M)
+	to_chat(M, "<span class ='cultlarge'><b>You hear faint chanting.. You feel a heavy weight upon your shoulders, as something shifts it's gaze towards you..</b></span>")
+	M.blur_eyes(30)
+	M.adjustOrganLoss(ORGAN_SLOT_BRAIN, 50, 60)
+	M.adjustStaminaLoss(50)
+	ADD_TRAIT(M, TRAIT_UNSTABLE, M)
+	//sanity = 75
+
+/datum/round_event/flutes/proc/flute_starlight(mob/living/carbon/M)
+	to_chat(M, "<span class='warning'><b>As the nearest stars light your skin and your station, you can make out the faint whispers being spoken in turn with the monotone flutes playing beyond you. You feel so tired, as the struts of metal piping, the walls, the floor twist in unnatural ways, as the lights dim.</b></span>")
+	sleep(300)
+	M.blur_eyes(40)
+	M.adjustOrganLoss(ORGAN_SLOT_BRAIN, 65, 70, 80)
+	ADD_TRAIT(M, TRAIT_UNSTABLE, M)
```

Comment:
Whats the point of the unstable trait if most people have mood disabled?

Reply:
It's for those that may infact have mood enabled.